### PR TITLE
Updated env var declarations to explicitly call out the local env val…

### PR DESCRIPTION
**Pull request by bdrainer**
…ue to use.

Updated the image name used by item-management-service.

It looks like we need to explicitly set the env var values.  There is a syntax that allows declaring the name and its matching local value using just the name but that syntax doesn't work in the 'x' extension fields.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,24 +29,23 @@ x-env:
   ITEMBANK_SERVICES_HISTORY_URL: http://host.docker.internal:8086
   SERVICE_ITEM_MANAGER_URL: http://host.docker.internal:8081
   SERVER_MAX_HTTP_HEADER_SIZE: 20000
-  SECURITY_ENABLED:
-  SESSION_CLUSTER_ENABLED:
```
Comment (bdrainer) [inline docker-compose.yml]: We don't need these two.

File: docker-compose.yml [hunk 2]
```diff
@@ -29,24 +29,23 @@ x-env:
...
 
 x-iat-env:
   &default-iat-env
-  SPRING_DATASOURCE_URL: jdbc:postgresql://host.docker.internal:5432/iat
-  DB_USER:
-  DB_PASSWORD:
-  IAT_AWS_ACCESS_KEY:
-  IAT_AWS_ACCESS_SECRET:
+  SPRING_DATASOURCE_URL: jdbc:postgresql://host.docker.internal:5432/iat  # this is overridden if config-service enabled
+  DB_URL: jdbc:postgresql://host.docker.internal:5432/iat # config-service application.yml uses DB_URL
```
Comment (bdrainer) [inline docker-compose.yml]: I had to put the DB_URL to get item management service to come up.  It worked without DB_URL when I did not have the config service running.  Once I added the config service item management failed to come up as it was trying to find Postgres at localhost.  Adding DB_URL fixed that.  The config-service uses the master branch from the iat-config-repo.  Make sure you pull the latest, but that is where you'll see in application.yml the DB_URL being used.  I feel it a bit odd that the SPRING_DATASOURCE_URL env var above didn't override the value even with the config service enabled.  Maybe my understanding of precedence is wrong.

Comment (malaffoon) [reply docker-compose.yml]: i really hope we can clean up all these properties and variables (looking at you @ftdholbrook)

**Comment by malaffoon** [comment]
Thanks Brett.

I'm uncertain about your comment about the env values. I did something like this:
```
x-test-env:
  &test-env
  FUBAR2:

my-service:
  environment:
    << : *test-env
    FUBAR3:
```
When i set values for FUBAR2 and FUBAR3 in my environment and then ran the docker-compose, both of them were set in my-service. What wasn't working for you?

File: docker-compose.yml (hunk 2 above)
Comment (ftdholbrook) [reply docker-compose.yml]: we should be able to do something like this
```
    volumes:
      - ${HOME}:/home
    environment:
      - SPRING_CONFIG_NAME=application,ap-iat-security
      - SPRING_CONFIG_LOCATION=/home/smarter_app/spring_confg/
```

File: docker-compose.yml (hunk 1 above)
Comment (malaffoon) [reply docker-compose.yml]: I've noticed SECURITY_ENABLED in a couple application.yml files ... can/should those be removed? For example, item-validation-service.

Comment (bdrainer) [reply docker-compose.yml]: Yes.  We need this put back.  I'll do that.

**Comment by bdrainer** [comment]
For reference, the error I'm getting when the DB_URL env var is not included in the compose file is:

`Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.apache.tomcat.jdbc.pool.DataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalArgumentException: URL must start with 'jdbc'`

**Comment by ftdholbrook** [comment]
@bdrainer / @malaffoon I am removing that DB_URL, and I think that what we are doing is slightly out of step with each other.  Pretty good odds the changes I am making will break this work, can we sync up and talk?

**Pull request merged**